packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/modules/DevLauncherInternalModule.kt

```diff
@@ -50,8 +50,6 @@ class DevLauncherInternalModule(reactContext: ReactApplicationContext?) :
 
   override fun getName() = "EXDevLauncherInternal"
 
-  override fun hasConstants(): Boolean = true
```

Comment:
unused code which is removed by upstream: https://github.com/facebook/react-native/commit/bbc3657ff4efd0218